Modelling/yourpage.py

```diff
@@ -0,0 +1,67 @@
+import tkinter as tk
+import pickle
+import tkinter.messagebox
+window = tk.Tk()
+window.title('Your main page')
+window.geometry('900x600')
+var1 = tk.StringVar()
+l = tk.Label(window, bg='yellow', width=16, textvariable=var1)
+l.pack()
+var1.set('Stock at hand')
+
+
+def print_selection():
+    value = lb.get(lb.curselection())
+
+    var1.set(value)
+
+
+b1 = tk.Button(window, text='print selection', width=15,
+               height=2, command=print_selection)
+b1.pack()
+var2 = tk.StringVar()
+# var2.set(('stock a', 'stock b', 'stock c', 'stock d'))
+lb = tk.Listbox(window, listvariable=var2, width = 46)
+
+# list_items=[1,2,3,4]
+# for item in list_items:
+#   lb.insert('end',item)
+# lb.insert(1, 'stock e')
+# lb.insert(2, 'stock f')
+# lb.insert(3, 'stock g')
+# lb.insert(4, 'stock h')
```

Comment:
This late in the timeline, code should not contain commented out lines. They should either be removed completely or uncommented.